src/_gettsim/transfers/erziehungsgeld.py

```diff
@@ -2,12 +2,18 @@
 Elterngeld."""
 from _gettsim.shared import add_rounding_spec, dates_active
 
+interpersonal_links_erziehungsgeld = {
+    "erziehungsgeld_eltern_m": {
+        "id_col": "p_id_erziehgeld_empf",
```

Comment:
```suggestion
        "id_col": "p_id_erziehungsgeld_empf",
```
We should never abbreviate within a word.

Reply:
I think we had this discussion before. This is due to the character limit (it's a basic input variable).
- `p_id_erziehgeld_empf` has exactly 20 characters
- `p_id_erziehungsgeld_empf` has 24 characters

One solution would be to use `p_id_erziehungsg_emp` but I have the feeling that `p_id_erziehgeld_empf` is easier to understand.